package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ mm/dd/17 richardjgowers, rathann, orbeckst, tylerjereddy, mtiberti, kain88-de,
   * 0.17.0
 
 Enhancements
+  * KDTree for neighbor search on periodic systems (PR #1660)
```

Comment:
I'm unsure if this is the correct way for the changelog. Please verify (same for testsuite/CHANGELOG)